specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -916,6 +1525,125 @@
           }
         }
       }
+    },
+    "AcsRouterChannelConfiguration": {
+      "description": "Router Channel Configuration",
+      "type": "object",
+      "properties": {
+        "channelId": {
+          "description": "Channel ID for Router Job",
+          "type": "string"
+        },
+        "capacityCostPerJob": {
+          "description": "Capacity Cost Per Job for Router Job",
+          "type": "integer",
+          "format": "int32"
+        },
+        "maxNumberOfJobs": {
+          "description": "Max Number of Jobs for Router Job",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "AcsRouterQueueInfo": {
+      "description": "Router Queue Info",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Router Queue Id",
+          "type": "string"
+        },
+        "name": {
+          "description": "Router Queue Name",
+          "type": "string"
+        },
+        "labels": {
+          "description": "Router Queue Labels",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "AcsRouterCommunicationError": {
+      "description": "Router Communication Error",
+      "properties": {
+        "code": {
+          "description": "Router Communication Error Code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Router Communication Error Message",
+          "type": "string"
+        },
+        "target": {
+          "description": "Router Communication Error Target",
+          "type": "string"
+        },
+        "innerError": {
+          "description": "Router Communication Inner Error",
+          "$ref": "#/definitions/AcsRouterCommunicationError"
+        },
+        "details": {
+          "description": "List of Router Communication Errors",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterCommunicationError"
+          }
+        }
+      }
+    },
+    "AcsRouterWorkerSelector": {
+      "description": "Router Job Worker Selector",
+      "type": "object",
+      "properties": {
+        "key": {
+          "description": "Router Job Worker Selector Key",
+          "type": "string"
+        },
+        "labelOperator": {
+          "description": "Router Job Worker Selector Label Operator",
+          "type": "string"
+        },
+        "value": {
+          "description": "Router Job Worker Selector Value",
+          "type": "object"
```

Comment:
This needs to reference a definition or use additionalProperties: true if it is meant to be a dictionary.

Reply:
no its just an object

Reply:
Then we need a definition. We don't allow anonymous types.

Reply:
I added it as AdditionalProperties

Reply:
I don't think this is correct based on the example which has an int value.

Reply:
this should be "type": "any"